op-service/txmgr/rpc.go

```diff
@@ -0,0 +1,88 @@
+package txmgr
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func NewTxmgrApi(txmgr TxManager, m metrics.RPCMetricer, l log.Logger) rpc.API {
```

Comment:
It is [conventional](https://go.dev/wiki/CodeReviewComments#initialisms) in Go to write acronyms in consistent capital (or small) letters, like
```suggestion
func NewTxmgrAPI(txmgr TxManager, m metrics.RPCMetricer, l log.Logger) rpc.API {
```